src/test/scala-2/chiselTests/BoringUtilsSpec.scala

```diff
@@ -34,12 +35,10 @@ class BoringUtilsSpec extends ChiselFlatSpec with ChiselRunners with Utils with
     BoringUtils.addSink(b, "x")
   }
 
-  behavior.of("BoringUtils.{addSink, addSource}")
+  behavior.of("BoringUtils.addSink and BoringUtils.addSource")
```

Comment:
Verilator or the Makefiles we generate have problems with some more exotic symbols. I need to circle back and replace these with simpler ones, like we do for whitespace.